### PR TITLE
dnf: add and reorder examples

### PR DESCRIPTION
**Pull request by jn64**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/dnf.md [hunk 1]
```diff
@@ -1,6 +1,19 @@
 # dnf
 
 > Package management utility for RHEL, Fedora, and CentOS (replaces yum).
+> More information: <https://dnf.readthedocs.io/>
+
+- Upgrade installed packages to newest available versions:
+
+`sudo dnf upgrade`
+
+- Search packages via keyword(s):
```
Comment (zdroid) [inline pages/linux/dnf.md:10]:
```suggestion
- Search packages via keywords:
```

**Comment by jn64** [comment]
I signed the CLA an hour ago.

**Comment by jn64** [comment]
@sbrl These commits were made on GitHub web, as you can see on the Verified tags on each commit, so the email is most definitely associated. However I'm using GitHub's private email option. I don't know how CLAassistant handles that.

File: pages/linux/dnf.md [hunk 2]
```diff
@@ -1,6 +1,19 @@
 # dnf
 
 > Package management utility for RHEL, Fedora, and CentOS (replaces yum).
+> More information: <https://dnf.readthedocs.io/>.
+
+- Upgrade installed packages to newest available versions:
```
[the remark this answers is not in the document]
Comment (jn64) [reply pages/linux/dnf.md:6]: Prefer 'the' or 'their' (as used in [apt](https://github.com/tldr-pages/tldr/blob/master/pages/linux/apt.md) and [apt-get](https://github.com/tldr-pages/tldr/blob/master/pages/linux/apt-get.md) pages)?

[the remark this answers is not in the document]
Comment (zdroid) [reply pages/linux/dnf.md:6]: `the` is simpler to grasp in this case.

**Pull request merged**